arduino/smartcar/smartcar.ino

```diff
@@ -11,21 +11,38 @@
 MQTTClient mqtt;
 WiFiClient net;
 
+
 // This is for the toggle button, to activate the safety features
 // This is changed to false to sync with the app better
-bool safetyFeatures = true;
 
-// stopZoneAutoBreak
+bool safetyFeatures = false;
+
+//stopZoneAutoBreak
 bool canDrive = true;
 bool driveForwards = true;
-bool driveBackwards = true;
-int speedGate; // check if it's positive or negative speed
-int theSpeed;  // lets the speed to actually be registered
+bool drivebackwards = true;
+int speedGate;//check if it's positive or negative speed
+int theSpeed;//let's speed actually be registered
 
-// incomingAvoidanceThreshold
+//incomingAvoidanceThreshold
 bool activeAvoidance = false;
 
-// controls which sensors active during the loops for the simulator car
+//registerCollision
+bool collision = false;
+int timeForCollision = 0;
+
+int angleBeforeUpdate;//needed for figure out angle change degree
+int angleAfterUpdate;
+
+int angleChangeDegree;
+int saveAngleOfChange;
+int checkAngleDegree;
+
+bool safeStationaryAngleChange = true;
+
+double carSpeed;
+
+//controls which sensors active during the loops for the simulator car
```

Comment:
Check spelling in comment

Reply:
I changed it to say simulated car first in the comment